### PR TITLE
The current Liquibase version is 3.8.4

### PR DESCRIPTION
**Pull request by szandany**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/liquibase.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Liquibase < Formula
   desc "Library for database change tracking"
   homepage "https://www.liquibase.org/"
-  url "https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.8.0/liquibase-3.8.0-bin.tar.gz"
+  url "https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.8.0/liquibase-3.8.4-bin.tar.gz"
```
Comment (bayandin) [inline Formula/liquibase.rb]: This URL doesn't exist

Comment (szandany) [reply Formula/liquibase.rb]: I corrected the link: https://github.com/liquibase/liquibase/releases/download/v3.8.4/liquibase-3.8.4.tar.gz

Comment (bayandin) [reply Formula/liquibase.rb]: It is required to update SHA256 accordingly

Comment (szandany) [reply Formula/liquibase.rb]: I updated the SHA256 accordingly.  Thanks for your help!

**Review by bayandin — APPROVED** [review]
Thanks @szandany 👍 
Also, you could use `brew bump-formula-pr` next time, it will help to update url and hash at once

**Pull request closed without merging**